teku/src/test/java/tech/pegasys/teku/cli/BeaconNodeCommandTest.java

```diff
@@ -306,7 +306,8 @@ private TekuConfigurationBuilder expectedConfigurationBuilder() {
         .setRestApiPort(5051)
         .setRestApiDocsEnabled(false)
         .setRestApiEnabled(false)
-        .setRestApiInterface("127.0.0.1");
+        .setRestApiInterface("127.0.0.1")
+        .setRestApiHostWhitelist(List.of("127.0.0.1", "localhost"));
```

Comment:
these should be defaults, did the tests not work without it? that would be a concern...

Reply:
this is building the "expected" config to compare the real one with